include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -47,14 +47,6 @@ class __reduce_mid_work_group_kernel;
 template <typename... _Name>
 class __reduce_kernel;
 
-// Storage helper since _Tp may not have a default constructor.
```

Comment:
Should we keep a comment similar to this one to clarify what this helper does? E.g., "Storage helper to support all types including ones without default constructor.".

Reply:
Yes, I added this comment back.